api/server.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+const express = require("express");
+const apiRouter = require("./routers");
+
+const port = 3000;
+const serverPath = `http://localhost:${port}`;
+const distPath = "dist/bootcamp/";
+
+const app = express();
+
+app.use(express.static(distPath));
+
+app.use("/api", apiRouter);
+
+app.all("*", (req, res) => {
+  res.sendFile(path.join(__dirname, "../" + distPath + "/index.html"));
```

Comment:
Better use join for every path element separated by comma

Reply:
fixed in the next PR, thanks